features/orphanage_sign_up.feature

```diff
@@ -29,3 +34,11 @@ Feature: As an orphanage
     And I fill in "Password confirmation" with "wrong_password"
     And I click "Sign up"
     Then I should see "Password confirmation doesn't match Password"
+
+  Scenario: User already exists
+    Given I am on the User registration page
```

Comment:
This step could be extracted to the background. It feels repetitive. Also 
```
     Given I am on the home page
      And I click "Register your orphanage"	
```
can easily be replaced in the first scenario